edb/schema/casts.py

```diff
@@ -153,7 +153,7 @@ def get_cast_fullname(
     from_type: s_types.TypeShell,
     to_type: s_types.TypeShell,
 ) -> sn.Name:
-    quals = [from_type.name, to_type.name]
```

Comment:
Hm, why does `.name` even work? Maybe disable it in favor of `.get_name()`?

Reply:
I assume this is for collection types, right?

Reply:
> Hm, why does `.name` even work? Maybe disable it in favor of `.get_name()`?

Historical.  It makes sense to remove direct attribute access, but I'd rather do in in a followup PR.